### DISCUSSION TITLE
Tab Card, working in HA Android App, but Not in a web page

### DISCUSSION THREAD

**Discussion by dCon0ne** [kinghat/tabbed-card#87] [Q&A]
So, I set up a Tab Card, with 4 tabs(3 in view), while editing the card, all tabs work the way they should.  Even the HA Android App., also work.  But does not in a chrome web browser.  
In my 4 tabs(3 in view) card, if I select the 3rd tab, the 3rd tab will move to the center of the card, exposing part of the 2nd tab on the left, and part of the 4th tab on the right.  

   BUT, if I select the 2nd tab, the 2nd tab **does not** center itself, or expose the 1st tab on the left.

![Tabcard issue](https://user-images.githubusercontent.com/22716511/219491848-b19a7a13-812d-4741-aa98-b3c9cf33a019.png)

Anyone else seeing this behaviour??   
Did I miss something in the Docs???

**Comment by kinghat** [comment]
i know its not obvious yet but you can scroll the tab bar horizontally with your mouse or finger. the behavior you are experiencing is probably just a combination of the unique width of your tabs and screen. can you please post your config for this card in case i look at it further?

**Reply by dCon0ne** [reply]
Thanks for getting back to me...   Yes, I can swipe right or left, when I have a Touch screen.  Is there a trick to using a mouse, I cant seem to get the mouse to work?

**Reply by kinghat** [reply]
you can see how the original component operates here: https://material-components.github.io/material-components-web-catalog/#/component/tabs there should be a scrollbar on hover but sadly i dont think thats true for chrome based browsers. eventually i will implement navigation arrows in the tab bar.
